spec/models/provider_ucas_preference_spec.rb

```diff
@@ -60,4 +60,18 @@
       expect(provider.ucas_preferences.gt12_response_destination).to eq new_email_address
     end
   end
+
+  describe 'application_alert_contact=' do
+    let(:provider) { create(:provider, ucas_preferences: ucas_preferences) }
+    let(:ucas_preferences) { build(:provider_ucas_preference) }
+    let(:new_email_address) { 'test@email.com' }
+
+    before do
+      provider.ucas_preferences.application_alert_contact = new_email_address
```

Comment:
it's not obvious from this diff what `provider` is, I suspect this is because of the way we set up the posts combined with the variable naming. I'll take a look